src/backend/distributed/commands/schema.c

```diff
@@ -128,6 +128,11 @@ PlanAlterObjectSchemaStmt(AlterObjectSchemaStmt *stmt, const char *queryString)
 			return PlanAlterFunctionSchemaStmt(stmt, queryString);
 		}
 
+		case OBJECT_EXTENSION:
+		{
```

Comment:
What about the other `ALTER EXTENSION` commands? 
We should have a higher level function like `PlanAlterExtensionStmt()`, in which we get into `PlanAlterExtensionSchemaStmt `.

And, in `PlanAlterExtensionStmt()` we handle others. As I've noted, we should give `NOTICE`, on `ADD/DROP` members since we're not going to implement them yet.

Also, we should plan/process ALTER EXTENSION UPDATE, which is probably the most important one

Reply:
I think, we should only have an entry for PlanAlterExtensionSchemaStmt in PlanAlterObjectSchemaStmt, not for the other kind of AlterExtension commands because of the code base structure.